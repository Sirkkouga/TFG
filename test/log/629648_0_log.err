INFO - 06/04/24 15:51:24 - 0:00:00 - Read 1071 words from the vocabulary file.

INFO - 06/04/24 15:51:24 - 0:00:00 - 920 words (1071 unique) in 8 sentences.
INFO - 06/04/24 15:51:24 - 0:00:00 - 43 unknown words (9 unique), covering 4.67% of the data.
INFO - 06/04/24 15:51:24 - 0:00:00 - ache: 19
INFO - 06/04/24 15:51:24 - 0:00:00 - im@@: 7
INFO - 06/04/24 15:51:24 - 0:00:00 - pti@@: 6
INFO - 06/04/24 15:51:24 - 0:00:00 - G: 3
INFO - 06/04/24 15:51:24 - 0:00:00 - List@@: 2
INFO - 06/04/24 15:51:24 - 0:00:00 - ger: 2
INFO - 06/04/24 15:51:24 - 0:00:00 - ach@@: 2
INFO - 06/04/24 15:51:24 - 0:00:00 - ak@@: 1
INFO - 06/04/24 15:51:24 - 0:00:00 - K: 1
INFO - 06/04/24 15:51:24 - 0:00:00 - Job completed successfully
INFO - 06/04/24 15:51:24 - 0:00:00 - Exiting after successful completion
