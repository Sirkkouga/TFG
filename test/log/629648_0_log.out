submitit INFO (2024-06-04 15:51:21,262) - Starting with JobEnvironment(job_id=629648, hostname=veuc09, local_rank=0(1), node=0(1), global_rank=0(1))
submitit INFO (2024-06-04 15:51:21,262) - Loading pickle: /home/usuaris/veu/javier.abella/TFG/test/log/629648_submitted.pkl
Saving the data to /home/usuaris/veu/javier.abella/TFG/test/java.test.sa.bpe.pth ...
submitit INFO (2024-06-04 15:51:24,313) - Job completed successfully
submitit INFO (2024-06-04 15:51:24,314) - Exiting after successful completion
